sysutils/angle-grinder/Portfile

```diff
@@ -20,13 +20,12 @@ long_description    Angle-grinder allows you to parse, aggregate, sum, \
                     your data in graphite/honeycomb/kibana/sumologic/splunk/etc. \
                     but still want to be able to do sophisticated analytics.
 
-depends_lib-append  path:lib/libssl.dylib:openssl
+checksums-append    ${name}-${version}${extract.suffix} \
```

Comment:
`${name}-${version}${extract.suffix}` can more simply be specified using `${distfiles}`.

Reply:
Would `${distfiles}` not be affected by all the crates?